src/commands/a32nx/fma.ts

```diff
@@ -0,0 +1,24 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const FMA_IMAGE_URL = 'https://media.discordapp.net/attachments/902990139670814750/1004144477717401671/unknown.png';
+
+export const fma: CommandDefinition = {
+    name: 'fma',
+    description: 'Provides a link to the FMA docs guide',
+    category: CommandCategory.A32NX,
+    executor: async (msg) => {
+        const fmaEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | Flight Mode Annunciator ',
+            description: makeLines([
+                'The Flight Mode Annunciator (FMA) at the top of the PFD, shows the status of the A/THR, the AP/FD vertical and lateral modes, the approach capabilities, and the AP, A/THR and FD engagement status.',
```

Comment:
```suggestion
                'The Flight Mode Annunciator (FMA) at the top of the PFD shows the status of the A/THR, the AP/FD vertical and lateral modes, the approach capabilities, and the AP, A/THR and FD engagement status.',
```

Tiny tiny correction, otherwise good to merge, I have fixed the conflict after merging #310 :)